src/bridge/MatrixHandler.ts

```diff
@@ -1030,7 +1005,7 @@ export class MatrixHandler {
         try {
             // Try to upload as a file and get URI
             //  (this could fail, see the catch statement)
-            contentUri = await this.ircBridge.uploadTextFile(fileName, text);
+            contentUri = await this.ircBridge.uploadTextFile(fileName, event.content.body);
```

Comment:
Not `ircAction.text`?

Reply:
`ircAction.text` is a modified version with non-printable characters for IRC formatting.
If we decide not to send it via IRC, we may as well upload the original Matrix body.

Reply:
Fair enough